### PR TITLE
feat: extend `auth` commands

### PR DESCRIPTION
**Pull request by dmvict**

### CODE REVIEW

File: x/auth/src/client/grpc.rs [hunk 1]
```diff
@@ -46,7 +54,9 @@ where
         let req = AuthNodeQueryRequest::Account(request.into_inner().try_into()?);
         let response = self.app.typed_query(req)?;
         let response: AuthNodeQueryResponse = response.try_into()?;
-        let AuthNodeQueryResponse::Account(response) = response;
+        let AuthNodeQueryResponse::Account(response) = response else {
+            return Err(Status::unknown("invalid response enum variant"));
```
Comment (joneskm) [inline x/auth/src/client/grpc.rs:58]: We usually return an `internal` error here e.g. https://github.com/rumos-io/gears/blob/abe901d3886b32481eeeea7e7c9796bb9e2805e6/x/bank/src/client/grpc.rs#L46-L48

File: x/auth/src/client/grpc.rs [hunk 2]
```diff
@@ -46,7 +54,9 @@ where
         let req = AuthNodeQueryRequest::Account(request.into_inner().try_into()?);
         let response = self.app.typed_query(req)?;
         let response: AuthNodeQueryResponse = response.try_into()?;
-        let AuthNodeQueryResponse::Account(response) = response;
+        let AuthNodeQueryResponse::Account(response) = response else {
+            return Err(Status::internal("invalid response enum variant"));
```
Comment (joneskm) [inline x/auth/src/client/grpc.rs:58]: I think it's better to be consistent and say "An internal error occurred while querying the application state."

File: core-types/src/query/response/auth.rs [hunk 3]
```diff
@@ -0,0 +1,3 @@
+pub use ibc_proto::cosmos::auth::v1beta1::QueryAccountResponse;
```
Comment (NYBACHOK) [inline core-types/src/query/response/auth.rs:1]: Are you sure that we should re-export this structures?

Comment (dmvict) [reply core-types/src/query/response/auth.rs:1]: The highlighted struct has been imported. I've added new structs and they are used in grpc handlers. I may skip reexporting but I have to import them directly.

File: x/auth/src/client/grpc.rs (hunk 1 above)
Comment (dmvict) [reply x/auth/src/client/grpc.rs]: done

**Pull request merged**